molo/surveys/templates/surveys/molo_survey_page_landing.html

```diff
@@ -18,7 +18,7 @@ <h4 class="surveys__intro">
 		<h3 class="surveys__question">{{ question }}</h3>
 		{% for answer, count in answers.items %}
 		  <div class="surveys__answer">
-			<span>{{ answer }}</span> {{ count }}
+			<span>{{ answer }}</span> {% if self.show_results_as_percentage%}{{ count }}%{% else %}{{count}}{% endif %}
```

Comment:
@saeedmb8 the count is either given back as a percentage or as a total, but the first count has a percentage sign next to it